src/index.html

```diff
@@ -17,16 +17,27 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      data-qa="big"
+      action="#"
+    >
+      <input
+        class="search_input_big form"
```

Comment:
Class names should represent the meaning of the content, not the styles. The class name 'search_input_big' (line 25) seems to describe the style rather than the content. Consider renaming it to reflect its function or content instead.